pom.xml

```diff
@@ -200,7 +200,14 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
+            <version>4.13.2</version>
```

Comment:
why? this is managed in the parent pom:
https://github.com/jenkinsci/plugin-pom/blob/620115c9dee7ddd9f8c8b42f968f44d54278c671/pom.xml#L186
```suggestion
```